assets/js/atomic/blocks/product-elements/rating/style.scss

```diff
@@ -66,15 +68,6 @@
 		.wc-block-components-product-rating {
 			@include placeholder();
 			width: 7em;
-			margin-left: auto;
-			margin-right: auto;
 		}
 	}
 }
-
-// Fix applied specifically to Classic Template
-.woocommerce-loop-product__link {
-	.wc-block-components-product-rating__stars {
-		display: block;
-	}
-}
```

Comment:
I'm a bit confused by this snippet and whether it's safe to remove it. Do you know what it was doing?

cc'ing @kmanijak since it was introduced recently in #7929.

Reply:
That fix was applied for a very unlikely case when Classic Template and Products block are used in the same page and Product Rating is inserted to Products block. In such a case [this filter is in use](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/src/BlockTypes/ProductRating.php#L78) and the "JS" stars are rendered.

Without the above, stars are inlined with price in Classic Template:
<img width="278" alt="image" src="https://user-images.githubusercontent.com/20098064/213676066-d048df5b-d213-4e0d-9d6d-dbccae377d62.png">

With this fix they are stacked, but with the margin change it doesn't center anymore:
<img width="278" alt="image" src="https://user-images.githubusercontent.com/20098064/213676236-19255008-59c4-4bec-ac42-6348c65b6a5c.png">

In my opinion:
1. I would keep this fix, since it's already there. And considering Classic Template will get removed pretty soon, so the fix will get removed (I assume we'll search through the code for "Classic Template's fixes" 😆)
2. I think nobody uses Classic Template AND Products block at the same page, so I don't think it's worth to spend time to fix the centring issue showed on the picture above. It seems so edge case.

Reply:
Oh, I see it now. Thanks for the details.

What do you think about implementing a different solution for this? I created a PR that would solve it without the need of that CSS snippet: https://github.com/woocommerce/woocommerce-blocks/pull/8247.

Reply:
Nice one! I'll go ahead and test as well, but looks promising!